google/ggrc-core
# GGRC-6616 Improve search of options placed in multiselect-dropdown

# Issue description

Implement option search by typing the name of needed option using keyboard.

# Steps to test the changes
Main areas for testing: global search, import and custom attributes.

1. Open multiselect-dropdown
2. Type the first letters of an existing option
**Expected result**: Detected option will be automatically highlighted
3. Type the first letters of a nonexistent option
**Expected result**: Previous highlighted option will remain highlighted
4. Press arrow down on the keyboard
**Expected result**: Next option will be highlighted
5. Press arrow up on the keyboard
**Expected result**: Previous option will be highlighted
6. Press enter on the keyboard
**Expected result**: Current  highlighted option will be checked/unchecked
7. Press escape on the keyboard
**Expected result**:  Multiselect dropdown will be closed

# Solution description
Implement option search by typing the name of needed option using keyboard. Arrow down/arrow up are used for highlighting next/previous option. Enter is used to check/uncheck current highlighted option. Escape closes the multiselect-dropdown.

# Sanity checklist


# PR Review checklist
